example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -41,49 +44,79 @@ describe('The S3 Ingest Granules workflow', () => {
     });
 
     it('has expected payload', () => {
-      expect(lambdaOutput.payload).toEqual(expectedPayload);
+      expect(lambdaOutput.payload).toEqual(expectedSyncGranulePayload);
     });
 
     it('has expected updated meta', () => {
-      expect(lambdaOutput.meta.input_granules).toEqual(expectedPayload.granules);
+      expect(lambdaOutput.meta.input_granules).toEqual(expectedSyncGranulePayload.granules);
     });
   });
 
-  describe('the PostToCmr Lambda', () => {
+  describe('the MoveGranules Lambda', () => {
     let lambdaOutput;
     let files;
     const existCheck = [];
 
     beforeAll(async () => {
-      lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'PostToCmr');
+      lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'MoveGranules');
       files = lambdaOutput.payload.granules[0].files;
       existCheck[0] = await s3ObjectExists({ Bucket: files[0].bucket, Key: files[0].filepath });
       existCheck[1] = await s3ObjectExists({ Bucket: files[1].bucket, Key: files[1].filepath });
       existCheck[2] = await s3ObjectExists({ Bucket: files[2].bucket, Key: files[2].filepath });
     });
 
     afterAll(async () => {
-      await s3().deleteObject({ Bucket: files[0].bucket, Key: files[0].name }).promise();
-      await s3().deleteObject({ Bucket: files[1].bucket, Key: files[1].name }).promise();
-      await s3().deleteObject({ Bucket: files[2].bucket, Key: files[2].name }).promise();
+      await s3().deleteObject({ Bucket: files[0].bucket, Key: files[0].filepath }).promise();
+      await s3().deleteObject({ Bucket: files[1].bucket, Key: files[1].filepath }).promise();
+      await s3().deleteObject({ Bucket: files[2].bucket, Key: files[2].filepath }).promise();
+    });
+
+    it('has a payload with updated filename', () => {
```

Comment:
Nice job with the descriptive test case names, particularly by contrast with those elsewhere in the file :)